packages/angular_devkit/build_angular/src/protractor/index.ts

```diff
@@ -81,7 +85,7 @@ export class ProtractorBuilder implements Builder<ProtractorBuilderOptions> {
         }
 
         // Compute baseUrl from devServerOptions.
-        if (options.devServerTarget && builderConfig.options.publicHost) {
+        if (builderConfig.options.publicHost) {
```

Comment:
This method is never called when `options.devServerTarget` is falsy

https://github.com/angular/angular-cli/blob/8c441d3f76d61d45076cfc2ef3fbcbf818422a14/packages/angular_devkit/build_angular/src/protractor/index.ts#L55